### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by saurikCornel** [osxfr33k/Asus-G750JX#2]
Thank you for your job. I use your clover a long time. Tell me please, do you have any plans about Big Sur? Thank you

**Comment by saurikCornel** [comment]
https://github.com/saurikCornel/Asus-g750jx-opencore-big-sur

**Comment by saurikCornel** [comment]
This is working big sur opencore. But without internal display and vga. Hope it helps someone. If you help fix vga and internal display would be great. Thank you!

**Comment by osxfr33k** [comment]
I have not had a chance to install BigSur yet, I may try it soon.  I also want to convert to OpenCore but need to make sure LCD brightness, HDMI etc does not break in OpenCore.   You were able to install BigSur with Clover Bootloader?  I just saw in the Rog Forum thread that Mitko has released a OpenCore EFI package and has installed BigSur successfully, I posted a reply over there.

Edited:  I just finished OpenCore 0.6.6 tested on Catalina not sure how it will work in Bad Sur :). Make sure you run GenSMBIOS to create your own Serial and UUID.  Look it up on the Dortania OpenCore guide you will find a link to it and read how to use it, very easy to use.
